src/Features/ERDDAP/Platform/Observations/CurrentConditions/wind.tsx

```diff
@@ -1,81 +1,163 @@
 /**
  * Wind specific current conditions card
  */
-import React from "react"
-import { Card, CardBody, CardHeader } from "reactstrap"
+import * as React from "react"
+import { Link } from "react-router-dom"
+import { Card, CardBody, CardHeader, Col } from "reactstrap"
 
 import { round } from "Shared/math"
 import { DataTimeSeries } from "Shared/timeSeries"
 import { compassDirection } from "Shared/unitConversion"
+import { useUnitSystem } from "Features/Units"
 import { UnitSystem } from "Features/Units/types"
 import { converter } from "Features/Units/Converter"
 
 import { WindTimeSeriesChart } from "components/Charts"
 
-import { PlatformDataset } from "../../../types"
+import { UseDatasets } from "../../../hooks"
+import { PlatformFeature, PlatformTimeSeries } from "../../../types"
+import { pickWindDatasets, pickWindTimeSeries } from "../../../utils/wind"
+import { cardProps, observationLink } from "./common_card"
 
-interface Props {
-  datasets: PlatformDataset[]
-  unit_system: UnitSystem
+interface WindCardProps {
+  platform: PlatformFeature
 }
 
-export const WindCard: React.SFC<Props> = ({ datasets, unit_system }) => {
-  if (datasets.length < 1) {
-    return null
+/**
+ * If there are wind time series for a platform load and display recent data.
+ *
+ * @param platform Platform GeoJSON feature to display wind data from
+ */
+export const WindCard: React.FunctionComponent<WindCardProps> = ({ platform }) => {
+  const unit_system = useUnitSystem()
```

Comment:
Codacy found an issue: [Identifier 'unit_system' is not in camel case.](https://app.codacy.com/manual/gmri/Neracoos-1-Buoy-App/pullRequest?prid=6166252)